.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,53 @@
+name: Feature request
+description: Create a report to help us fix something that isn't working as expected
+title: "[Feature]"
+labels: "feature request :mailbox_with_mail:"
+body:
+- type: markdown
+  attributes: 
+    value: |
+      ## Before you begin
+      🚨 **Please do not skip instructions**🚨
+      
+      This info is essential to help us understand your feature request.
+      
+      If you're uncertain about the feature, find or start an [Ideas](https://github.com/CommunityToolkit/WindowsCommunityToolkit/discussions/categories/ideas) discussion, where you can get insight from the community regarding your issue. 
+- type: textarea
+  id: problem-description
+  validations: 
+    required: true
+  attributes: 
+    label: Describe the problem
+    description: Describe the problem this feature would solve.
+    placeholder: |
+       A clear and concise description of what the problem is, starting with the user story.
+
+       Describe or link to any existing issues or discussions.
+       
+       Provide examples of the restrictions in the current environment that hinders the work your users or you want to perform. What are the ways this new feature will help transform and deliver those results
+       
+       For example, I am currently using the InfiniteCanvas control which lacks the TabbedCommandBar control feature. I am looking to improve user experience therefore i would like to use that in my project to provide ease of accessibility and a user-friendly interface. This new feature will provide quick access to the toolbar, enhance space utilization, etc [...]
+
+- type: textarea
+  id: solution-description
+  validations: 
+    required: true
+  attributes: 
+    label: Describe the solution
+    description: Describe the solution to the problem.
+    placeholder: |
+       A clear and concise description of what you want to happen. Define how do you think it will help the community and the value it adds to the toolkit.
+
+- type: textarea
+  id: alternatives
+  attributes: 
+    label: Alternatives
+    description: Describe alternatives you've considered
+    placeholder: |
+       A clear and concise description of any alternative solutions or features you've considered.
+
+- type: textarea
+  id: additional-info
+  attributes:
+    label: Additional info
+    description: Add any other context or screenshots about the feature request here.
```

Comment:
Could be cool now to have a checkbox to ask if they'd like to implement the feature themselves (same for bugs)?

Reply:
It would be! Should I add this in and see how it looks now?

Reply:
On second thought, perhaps this is better handled by GitHub itself so it stays up to date.

Reply:
Ah, I mean the context here is that some folks are only interested in filing issues and not necessarily contributing, but we don't always know that without asking, so it's a common thing we ask frequently. Otherwise it can put a lot of burden on us to just have issues sit around if we can't prioritize them.

So, I was thinking if we just had an initial checkbox at the bottom of bugs/features that asks if they're interested in contributing, collaborating, or working on the item from the start it's save some of that interaction/turn-around time, and we'd know right away. (Or can we do radio boxes?) Something like this:
![image](https://user-images.githubusercontent.com/24302614/144675580-6bcaa568-e1d9-4bbf-90c8-c7fd68ba2341.png)

If there has to be a default maybe we need a 4th one that's "No Response"...?

Reply:
I see. We can do that, it'll show as a set of checkboxes where they can pick one or none.